lib/events/dynamoevents/dynamoevents.go

```diff
@@ -204,6 +237,28 @@ func New(cfg Config) (*Log, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	// Enable continuous backups if requested.
+	if b.Config.EnableContinuousBackups {
+		if err := dynamo.SetContinuousBackups(context.TODO(), b.svc, b.Tablename); err != nil {
```

Comment:
I would consider passing context to the New() method and use here and below so these calls can be properly canceled.